app/src/main/res/values/themes.xml

```diff
@@ -1,20 +1,12 @@
-<resources xmlns:tools="http://schemas.android.com/tools">
-    <!-- Base application theme. -->
-    <style name="Base.Theme.Erabook" parent="Theme.Material3.Light.NoActionBar">
-        <!-- Customize your light theme here. -->
-        <!-- <item name="colorPrimary">@color/my_light_primary</item> -->
+<resources>
+    <style name="Base.Theme.Erabook" parent="Theme.MaterialComponents.DayNight.NoActionBar">
         <item name="colorPrimary">@color/primary</item>
         <item name="colorSecondary">@color/secondary</item>
-        <item name="backgroundColor">@color/lightb</item>
+        <item name="backgroundColor">@color/secondary</item>
         <item name="textFillColor">@color/black</item>
+        <item name="android:textColor">@color/black</item>
     </style>
 
     <style name="Theme.Erabook" parent="Base.Theme.Erabook" />
 
-
-<!--    <style name="moreScreenDividerStyle">-->
-<!--        <item name="android:layout_width">0dp</item>-->
-<!--        <item name="android:layout_height">1dp</item>-->
-<!--        <item name="background">#989494</item>-->
```

Comment:
Please delete commented lines.

Reply:
this one shows that these are the deleted lines

Reply:
You're right, I can see it now 😆